CMakeLists.txt

```diff
@@ -106,7 +106,9 @@ option(IREE_ENABLE_EMITC "Enables MLIR EmitC dependencies." ${IREE_BUILD_COMPILE
 set(IREE_ALL_HAL_DRIVERS
```

Comment:
general note: I avoid mentioning github issues in _commit descriptions_ and only include them in _PR descriptions_ to avoid this spam (particularly relevant when iterating a lot and force pushing): 
![image](https://user-images.githubusercontent.com/4010439/134944308-31cc05d9-13c6-40e4-9296-57c828f7a746.png)

Reply:
ahh.... Sorry about that. duly noted.